Library/Formula/infer.rb

```diff
@@ -0,0 +1,60 @@
+class Infer < Formula
+  desc "Facebook Infer is a static analysis tool - if you give Infer some Objective-C, Java, or C code, it produces a list of potential bugs."
```

Comment:
Do you mind shortening this description? "infer: DESCRIPTION" needs to fit in 80 characters.